kolibri/plugins/coach/assets/src/views/create-exam-page/index.vue

```diff
@@ -125,7 +126,8 @@
       numQuestions: 'Number of questions',
       enterNum: 'Enter a number',
       examRequiresTitle: 'The exam requires a title',
-      examRequiresNum: 'The exam requires a number of questions between 1 and 50',
+      numQuestionsBetween: 'The exam requires a number of questions between 1 and 50',
```

Comment:
Could we maybe make the final number dynamic, depending on the smaller of 50 or the number of questions available?